src/welcome/getting-started.js

```diff
@@ -0,0 +1,164 @@
+
+/**
+ * Internal dependencies
+ */
+import SVGTutorialsIcon from './images/tutorials.svg'
+import SVGDocsIcon from './images/docs.svg'
+import SVGCommunityIcon from './images/user.svg'
+import SVGArrowUpRightIcon from './images/arrow-up-right.svg'
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n'
+import { Fragment } from '@wordpress/element'
+
+/**
+ * External dependencies
+ */
+import { i18n } from 'stackable'
+
+const generalProps = {
+	label: 'general',
+	content: [
+		{
+			title: __( 'Tutorials', i18n ),
+			subtitle: __( 'Get to know the plugin and start your journey with our brand new Stackable Courses.', i18n ),
+			link: '#',
+			icon: <SVGTutorialsIcon />,
+		},
+		{
+			title: __( 'Documentation and Support', i18n ),
+			subtitle: __( 'Get assistance with troubleshooting, guides, FAQs, and updates.', i18n ),
+			link: '#',
+			icon: <SVGDocsIcon />,
+		},
+		{
+			title: __( 'Community', i18n ),
+			subtitle: __( 'Join our very active Stackable Community on Facebook.', i18n ),
+			link: '#',
+			icon: <SVGCommunityIcon />,
+		},
+	],
+}
+
+const essentialsProps = {
+	label: 'essentials',
+	content: [
+		{
+			title: __( 'How to Add Blocks', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Customize Blocks', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Use Copy & Paste Styles', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'Introduction to Wireframes', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Use the Design Library', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Use the Column Blocks', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+	],
+	headerTitle: __( 'Learn the essentials', i18n ),
+	headerSubtitle: __( 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Consequat tempus sollicitudin rhoncus pretium tellus.', i18n ),
+}
+
+const Section = props => {
+	const Card = ( {
+		title, subtitle, link, icon = '', src = '',
+	} ) => {
+		return <div className="s-card">
+			{ props.label === 'general'
```

Comment:
For "general" cards, please make the entire card the link, and then please add a transition to make the shadow larger when hovering over the card. On hover, please transition to:

```css
box-shadow: rgba(0, 0, 0, 0.16) 0px 10px 36px 0px, rgba(0, 0, 0, 0.06) 0px 0px 0px 1px;
```